Managers/WindowMan.h

```diff
@@ -178,12 +191,12 @@ namespace RTE {
 		/// <param name="newResY">New height to resize to.</param>
 		/// <param name="upscaled">Whether the new resolution should be upscaled.</param>
 		/// <param name="displaysAlreadyMapped">Whether to skip mapping displays because they were already mapped elsewhere.</param>
-		void ChangeResolution(int newResX, int newResY, bool upscaled, bool displaysAlreadyMapped = false);
+		void ChangeResolution(int newResX, int newResY, float newResMultiplier = 1.0f, bool fullscreen = false, bool displaysAlreadyMapped = false);
 
 		/// <summary>
-		/// Switches to a new resolution multiplier.
+		/// Toggles between 1x and 2x resolution multiplier.
 		/// </summary>
-		void ChangeResolutionMultiplier();
+		void ToggleResolutionMultiplier();
```

Comment:
Do we still need the option for a resolution multiplier? If we can arbitrarily scale the window, what's the point - can't we just always render at the given resolution but then in fullscreen modes always scale to the monitor size?

Reply:
It's not necessarily relevant for fullscreen, but for windowed modes it somewhat simplifies the settings.
And user side imo a resolution multiplier/zoom level is much easier to understand.
 Also for accessibility having just one fixed resolution is not the best.
 If we could independently scale the GUI it might be fine.

Reply:
The toggle could be switched to be just fullscreen or no fullscreen (so long as it's single screen fullscreen, multi display letterboxing is beyond me rn)